[DOC] pprof endpoint fix in exporters (PMM-14686) (#5067)
percona/pmm: documentation/docs/troubleshoot/exporter_issues.md

# Exporter issues

## Exporter consuming excessive CPU or memory

If a PMM exporter (such as `mysqld_exporter`, `postgres_exporter`, or `mongodb_exporter`) is using excessive CPU or memory, you can collect profiling data to diagnose the issue.

PMM exporters expose `/debug/pprof/` endpoints for performance profiling. Use these to generate diagnostic data for analysis or to share with Percona Support.

### Collect profiling data

The `pprof` endpoints use basic authentication:

- **username:** `pmm`
- **password:** The exporter's `agent_id` (default), or your custom password if configured

To collect a profile, you need the exporter's password and listening port:
{.power-number}

> 1. Find the exporter's listening port and `agent_id` (used as the default password):

Reviewer: ```suggestion
1. Find the exporter's listening port and password (`agent_id` or your custom password, if set up):
```

```bash
    pmm-admin list
```

2. From the PMM Client host, collect the profile:
```bash
    # Heap (memory) profile
    curl -su pmm:<agent_id> http://127.0.0.1:<port>/debug/pprof/heap > heap.pprof

    # CPU profile (60 seconds)
    curl -su pmm:<agent_id> "http://127.0.0.1:<port>/debug/pprof/profile?seconds=60" > cpu.pprof
```

>     Replace `<agent_id>` and `<port>` with values from `pmm-admin list`.

Reviewer: I would replace all mentions of agent_id by password.
